fabric_am/handlers/net_handler.py

```diff
@@ -904,8 +904,16 @@ def __portmirror_create_data(self, sliver: NetworkServiceSliver, service_name: s
         if sliver.mirror_direction:
             direction = str(sliver.mirror_direction).lower()
         data = {"name": service_name,
-                "from-interface": self.parse_interface_name(sliver.mirror_port),
                 "direction": direction}
+        interface = self.parse_interface_name(sliver.mirror_port)
+        if "outervlan" in interface or "innervlan" in interface:
+            if 'from-interface-vlan' not in data:
+                data['from-interface-vlan'] = []
+            data['from-interface-vlan'].append(self.parse_interface_name(sliver.mirror_port))
+        else:
+            if 'from-interface' not in data:
+                data['from-interface'] = []
+            data['from-interface'].append(self.parse_interface_name(sliver.mirror_port))
```

Comment:
Please use `sliver.mirror_vlan` for vlan tag.

Reply:
Changed.